docs/installation.rst

```diff
@@ -98,6 +98,16 @@ On Ubuntu systems:
     export PATH=$PATH:~/.local/bin
     pip3 install ansible ansible-rulebook ansible-runner
 
+On macOS systems:
+
+.. code-block:: shell
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
I since it might be several ways to install JVM in mac I would separate this snippet and leave clear that the use of homebrew is just a suggestion.